stable/grafana/README.md

```diff
@@ -91,6 +91,8 @@ The command removes all the Kubernetes components associated with the chart and
 | `sidecar.resources`              | Sidecar resources | `{}`       |
 | `sidecar.dashboards.enabled`              | Enabled the cluster wide search for dashboards and adds/updates/deletes them in grafana | `false`       |
 | `sidecar.dashboards.label`                | Label that config maps with dashboards should have to be added | `grafana_dashboard`                                |
+| `sidecar.dashboards.folder`                | Folder in the pod that should hold the collected dashboards (unless `defaultFolderName` is set). This path will be mounted. | `/tmp/dashboards`    |
+| `sidecar.dashboards.defaultFolderName`                | he default folder name, it will create a subfolder under the `folder` and put dashboards in there instead | `nil`                                |
```

Comment:
if this creates a subfolder under `folder`, can't we just use `folder` to specify the full path?

Reply:
if we use the `folder`, sidecar will mount just that folder, and will not get correct permissions to the parent folder. When scraping and saving the dashboards happen, it will error with permission issue.

Reply:
@maorfr Are we okay with this? :)

Reply:
i think we are :)